src/views/Generator/Generator.utils.ts

```diff
@@ -31,12 +31,6 @@ export async function exportScript(fileName: string) {
   await window.studio.script.saveScript(script, fileName)
 }
 
-export const scriptExists = async (fileName: string) => {
-  return window.studio.ui
-    .getFiles()
-    .then((files) => files.scripts.includes(fileName))
-}
```

Comment:
This is not needed, because we always have the current list of files in the ui store